site/src/sphinx/server-annotated-service.rst

```diff
@@ -0,0 +1,627 @@
+.. _@Get: apidocs/index.html?com/linecorp/armeria/server/annotation/Get.html
+.. _@Head: apidocs/index.html?com/linecorp/armeria/server/annotation/Head.html
+.. _@Options: apidocs/index.html?com/linecorp/armeria/server/annotation/Options.html
+.. _@Order: apidocs/index.html?com/linecorp/armeria/server/annotation/Order.html
+.. _@Param: apidocs/index.html?com/linecorp/armeria/server/annotation/Param.html
+.. _@Patch: apidocs/index.html?com/linecorp/armeria/server/annotation/Patch.html
+.. _@Path: apidocs/index.html?com/linecorp/armeria/server/annotation/Path.html
+.. _@Post: apidocs/index.html?com/linecorp/armeria/server/annotation/Post.html
+.. _@ProduceType: apidocs/index.html?com/linecorp/armeria/server/annotation/ProduceType.html
+.. _@Put: apidocs/index.html?com/linecorp/armeria/server/annotation/Put.html
+.. _@RequestConverter: apidocs/index.html?com/linecorp/armeria/server/annotation/RequestConverter.html
+.. _@RequestObject: apidocs/index.html?com/linecorp/armeria/server/annotation/RequestObject.html
+.. _@ResponseConverter: apidocs/index.html?com/linecorp/armeria/server/annotation/ResponseConverter.html
+.. _@Trace: apidocs/index.html?com/linecorp/armeria/server/annotation/Trace.html
+.. _AggregatedHttpMessage: apidocs/index.html?com/linecorp/armeria/common/AggregatedHttpMessage.html
+.. _ByteArrayRequestConverterFunction: apidocs/index.html?com/linecorp/armeria/server/annotation/ByteArrayRequestConverterFunction.html
+.. _DecoratingServiceFunction: apidocs/index.html?com/linecorp/armeria/server/DecoratingServiceFunction.html
+.. _ExceptionHandlerFunction: apidocs/index.html?com/linecorp/armeria/server/annotation/ExceptionHandlerFunction.html
+.. _HttpResponse: apidocs/index.html?com/linecorp/armeria/common/HttpResponse.html
+.. _HttpResponseException: apidocs/index.html?com/linecorp/armeria/server/HttpResponseException.html
+.. _HttpStatusException: apidocs/index.html?com/linecorp/armeria/server/HttpStatusException.html
+.. _JacksonRequestConverterFunction: apidocs/index.html?com/linecorp/armeria/server/annotation/JacksonRequestConverterFunction.html
+.. _PathMapping: apidocs/index.html?com/linecorp/armeria/server/PathMapping.html
+.. _RequestConverterFunction: apidocs/index.html?com/linecorp/armeria/server/annotation/RequestConverterFunction.html
+.. _ResponseConverterFunction: apidocs/index.html?com/linecorp/armeria/server/annotation/ResponseConverterFunction.html
+.. _ServerBuilder: apidocs/index.html?com/linecorp/armeria/server/ServerBuilder.html
+.. _Service: apidocs/index.html?com/linecorp/armeria/server/Service.html
+.. _StringRequestConverterFunction: apidocs/index.html?com/linecorp/armeria/server/annotation/StringRequestConverterFunction.html
+
+.. _server-annotated-service:
+
+Annotated HTTP Service
+======================
+
+*Armeria* provides a way to write an HTTP service using annotations. It helps a user to make his or her codes
```

Comment:
- `*Armeria*` -> `Armeria`? `*Armeria*` was used in the front page since I was defining a new term that appeared first time.
- help a user to make -> help a user make ('to' is not necessary for 'help')
- codes -> code (code is uncountable.)